src/client/src/orangehrmCorporateDirectoryPlugin/pages/CorporateDirectory.vue

```diff
@@ -72,7 +72,12 @@
     <div :class="{'orangehrm-corporate-directory': hasCurrentIndex}">
       <div class="orangehrm-paper-container">
         <table-header :show-divider="false" :total="total"></table-header>
-        <oxd-grid ref="scrollerRef" :cols="colSize" class="orangehrm-container">
+        <oxd-grid
+          ref="scrollerRef"
+          :cols="colSize"
+          class="orangehrm-container"
```

Comment:
lets move class to a computed property. check how it's done here https://github.com/orangehrm/oxd/blob/287d99973aaeb825c97392be91b7b62b33a5fc07/components/src/core/components/Toast/Toast.vue#L23

Reply:
Done.